### PR TITLE
[CI] fix bugs for multigpu benchmarks

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Description

Arg `low_mem` has already been removed from `dgl.nn.pytorch.RelGraphConv`, let's just remove it from benchmark tests as well.
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by Rhett-Ying** [comment]
@dgl-bot run g4dn.12xlarge multigpu.bench_multigpu dmlc/0.9.x

File: benchmarks/benchmarks/utils.py [hunk 1]
```diff
@@ -534,7 +534,7 @@ def skip_if_not_4gpu():
     """skip if DGL_BENCH_DEVICE is gpu"""
 
     def _wrapper(func):
-        if GPU_COUNT != 4:
+        if GPU_COUNT < 4:
```
Comment (jermainewang) [inline benchmarks/benchmarks/utils.py:537]: Why this change?

**Review by jermainewang — APPROVED** [review]
Could you briefly describe the changes?

**Comment by frozenbugs** [comment]
Why merge directly to 1.0.x instead of master?

File: benchmarks/benchmarks/utils.py (hunk 1 above)
Comment (Rhett-Ying) [reply benchmarks/benchmarks/utils.py:537]: because I think any instance with no less than 4 GPUs can run these tests, no need to be exactly equal `4`.

**Comment by Rhett-Ying** [comment]
I'd like to verify on `1.0.x` and I think it's ok to cherry-pick to `master`.

**Comment by Rhett-Ying** [comment]
> Could you briefly describe the changes?

added into description.

**Comment by Rhett-Ying** [comment]
@dgl-bot run g4dn.12xlarge multigpu.bench_multigpu dmlc/0.9.x

**Review by frozenbugs — APPROVED** [review]

**Comment by Rhett-Ying** [comment]
@dgl-bot run g4dn.12xlarge multigpu.bench_multigpu dmlc/0.9.x

**Comment by Rhett-Ying** [comment]
@dgl-bot run g4dn.12xlarge model_acc.bench_gat dmlc/0.9.x

**Comment by Rhett-Ying** [comment]
This PR is pending on https://github.com/dmlc/dgl/pull/5134. Need to rebase on it to verify multigpu benchmarks work as expected.

**Comment by Rhett-Ying** [comment]
this has been covered in https://github.com/dmlc/dgl/pull/5140. and `1.0.x` has been reset to latest master. let's close this PR.
